syntax = "proto2";

import "header.proto";
import "pose.proto";

package simple_msgs;

message PoseStamped {
  required simple_msgs.Header header = 1;
  required simple_msgs.Pose pose = 2;
}